nova/requirements.yaml

```diff
@@ -2,6 +2,9 @@ dependencies:
   - name: postgres
     repository: http://localhost:8879/charts
     version: 0.1.0
+  - name: rabbitmq-notifications
+    repository: http://localhost:8879/charts
+    version: 0.0.1
```

Comment:
Could you add a condition to the dependency, so that the rabbitmq is not included, if not enabled?
See [here](https://github.com/kubernetes/helm/blob/master/docs/charts.md#tags-and-condition-fields-in-requirementsyaml).

Also, wouldn't it possible to define an [alias](https://github.com/kubernetes/helm/blob/master/docs/charts.md#alias-field-in-requirementsyaml) on the chart `rabbitmq`, and then make the chart derive the variables dynamically, and avoid the duplication by that?

Reply:
Good point! I wasn't aware that templates are supported for requirements - but indeed it works and so I can make this invisible.

Do you have an example for the second proposal? Ich stehe auf dem Schlauch

Reply:
Sure, the idea is, that we already have a `rabbitmq` chart. The issue is, that it will create only create a fixed name deployment and service.

Instead of creating a new chart, I think you can change the chart to derive the name for service and pod from the chart (e.g.`.Chart.Name`, and then do something like:
```
  - name: rabbitmq
    alias: rabbitmq-notifications
    repository: http://localhost:8879/charts
    version: 0.0.1
```
Haven't tried it myself, but as I understand it, that is how it could work.

If that doesn't work, it should give you at least a different namespace,
so, given an alias of `rabbitmq_notifications`, you could introduce a value `suffix` and set it:
```#values.yaml
rabbitmq_notifications:
     suffix: "-notifications"
```
And add  the suffix to service and deployment.

Reply:
I believe we should do this when the RPCs are separated per service, after those have moved to the helm-charts repo.

I would not delay merging this PR, should I?

Reply:
I don't see, why it has to wait until the RPCs are being separated. 

I want to avoid to have two rabbitmq charts, and I think, it isn't a major effort to do so.

Reply:
the point is that I used @ruvr 's helm-charts/common/rabbit as a starting point, not the openstack-helm/rabbitmq

I will of consolidate with that one once nova, neutron, cinder, glance move to helm-charts. All that copy&paste has never been meant to stay longer than openstack-helm exists.

Reply:
PR is updated